include/zephyr/net/wifi.h

```diff
@@ -27,6 +27,18 @@
 #define WIFI_LISTEN_INTERVAL_MIN 0
 #define WIFI_LISTEN_INTERVAL_MAX 65535
 
+/**
```

Comment:
It would be better to document this in the structure below, as this describes the parameter usage and not just the ranges.